config/config.exs

```diff
@@ -8,7 +8,8 @@
 use Mix.Config
 
 config :rebay_api,
-  ecto_repos: [RebayApi.Repo]
+  ecto_repos: [RebayApi.Repo],
+  client_host: System.get_env("CLIENT_HOST")
```

Comment:
Am I correct in assuming this is used to store where the active service is running? What does `CLIENT` indicate here?

Reply:
This is a bit of an ugly side-effect from our authentication troubles. `CLIENT_HOST` is the host path of the frontend app for our session controller to respond to after login. At that point in the authentication flow, there's no information [that we could find] that we can pull out of the request to the same effect.

Reply:
Essentially, we were struggling to come up with a good way to send the cookie back to the frontend in the authentication flow.

Reply:
Ah right on, I wasn't understanding this correctly then.